utils/python/CIME/XML/env_batch.py

```diff
@@ -32,23 +32,60 @@ def set_value(self, item, value, subgroup=None):
 
         return val
 
-    def get_value(self, item, attribute={}, resolved=True, subgroup=None):
-        value = None
-        nodes = self.get_nodes("entry",{"id":item})
-        if len(nodes) > 0:
-            value = {} # Not consistent with return values for other classes' get_value methods
-            if subgroup is None:
-                nodes = self.get_nodes("job")
-            else:
-                nodes = self.get_nodes("job", {"name":subgroup})
+    # def get_value(self, item, attribute={}, resolved=True, subgroup=None):
+#         value = None
+#         logger.debug("Value=%s additional Attribute:%s" , item , attribute)
+#         nodes = self.get_nodes("entry",{"id":item})
+#         logger.debug("Nodes:%s" , nodes)
+#         if len(nodes) > 0:
```

Comment:
Maybe we should have get_value in env_batch - it would make the subgroup argument required.   Or maybe we don't need it?